### PR TITLE
Enhanced the Translator App as well as added a new logo

### PR DESCRIPTION
**Pull request by kanisshka**
## Related Issue
Minion image is not visible as well as some enhancement is needed.

Closes: #1071 

### Describe the changes you've made
Attached screenshot related to this.

### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA

### Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] New and existing unit tests pass locally with my changes.

### Screenshots
Screenshot (627)

LOGO
image

### CODE REVIEW

**Comment by kanisshka** [comment]
Please have a look.. @Vishal-raj-1 @urvashi-code1255 @ridsuteri @harshgupta20

File: public/translatorMinions.html [hunk 1]
```diff
@@ -1,32 +1,34 @@
...
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">       
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Translator App</title>
+    <link rel="stylesheet" href="../assets/css/minions.css"/>  
+    <link rel="icon" href="https://www.fivesquid.com/pics/t2/1465460352-55810-1-1.png" type="image/x-icon">
+    </head>
+=======
```
Comment (ridsuteri) [inline public/translatorMinions.html:10]: does this ====== has a purpose? 
if not please remove it

**Review by ridsuteri — CHANGES_REQUESTED** [review]
@kanisshka looks fine to me , but this project is saying some error with server when i am trying to translate on my side...
kindly check once again locallly and there is a minor correction required in the html file

File: public/translatorMinions.html (hunk 1 above)
Comment (kanisshka) [reply public/translatorMinions.html:10]: I didn't add this on my side but still its only commits,
I remove this Sorry for this.

**Comment by kanisshka** [comment]
I am making new PR for this.

**Pull request closed without merging**